package.json

```diff
@@ -41,6 +41,7 @@
   },
   "scripts": {
     "karma": "karma start vision/explore-api/karma.conf.js",
+    "karma": "karma start speech/explore-api/karma.conf.js",
```

Comment:
can't have two lines with the same key, change the origin `karma` line to:

    "karma": "karma start vision/explore-api/karma.conf.js && karma start speech/explore-api/karma.conf.js",